src/ResourceManagement/Resource/Microsoft.Azure.Management.V2.Resource/Core/CreatableUpdatableResourcesRoot.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.V2.Resource.Core.ResourceActions;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Management.V2.Resource.Core
+{
+    /// <summary>
+    /// Implementation of <see cref="ICreatableUpdatableResourcesRoot{IFluentResourceT}">
+    /// </summary>
+    /// <typeparam name="IFluentResourceT">the type of resources in the batch</typeparam>
+    internal class CreatableUpdatableResourcesRoot<IFluentResourceT> :
+        CreatableUpdatable<ICreatableUpdatableResourcesRoot<IFluentResourceT>,
+            object,
```

Comment:
`CreatableUpdatableResourcesRoot` is internal and is used as dummy local resource that act as parent node for real creatable resources in the graph. It's extends from `CreatableUpdatable` so that it can place itself into graph.

So it does not have a inner type instance (second generic param to CreatableUpdatable) and is not updatable (last generic param to CreatableUpdatable). 

But looks like we should be able to get rid of first `object`, will update the PR.